rabbitmq/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # CHANGELOG - rabbitmq
 
 
+1.4.0 / 2017-09-04
+==================
+
+### Changes
+
+* [BUGFIX] Add a key check before updating connection state metric.
```

Comment:
Could you also add a link to the PR at the end of the line and the end of the file. You have example on how to do it line 17 or 18.